industrial_extrinsic_cal/launch/calibrationTarget.py

```diff
@@ -0,0 +1,122 @@
+# This script provides functions to generate an image to be used as a target

```

Comment:
Jeremy Zoss created this for printing targets. MPix.com and fractureme.com can print very nice targets on either metal, glass or acrylic